amd64-usbtpu.Dockerfile

```diff
@@ -76,24 +84,17 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         build-essential \
         libedgetpu1-std \
     && rm -rf /var/lib/apt/lists/* \
-    && python3 -m pip install --upgrade pip setuptools==41.0.0 wheel && pip install \
-        aiofiles \
-        fastapi \
-        uvicorn \
-        pyhumps \
-        pytest \
+    && python3 -m pip install --upgrade pip setuptools==41.0.0 wheel && pip install -r /requirements.txt \
 	https://dl.google.com/coral/python/tflite_runtime-2.1.0.post1-cp37-cp37m-linux_x86_64.whl \
     && apt-get purge -y \
         build-essential \
         python3-dev \
+        pyhumps \
```

Comment:
I guess pyhumps shouldn't be in this _apt purge_ line. it gives "E: Unable to locate package pyhumps" error.